### PR TITLE
Merge Flow and server logs for containerized execution.

### PR DESCRIPTION
**Pull request by djaiswal83**
Merge Flow and server logs for containerized execution.
Some code cleanup in FlowContainer around EventReporter.

### CODE REVIEW

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java [hunk 1]
```diff
@@ -635,19 +637,32 @@ public static void deleteSymlinkedFile(final Path symlinkedFilePath)
     }
   }
 
+  /**
+   * Uploads the log file to the db for persistence.
+   * @param execId execution id of the flow.
+   */
+  private void uploadLogFile(final int execId) {
+    try {
+      this.executorLoader.uploadLogFile(execId, "", 0, FlowContainer.logFile);
```
Comment (aditya1105) [inline azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java:646]: Why the name of the log file is emptyString?

**Review by aditya1105 — APPROVED** [review]

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java (hunk 1 above)
Comment (djaiswal83) [reply azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java:646]: The name is only used for job log as per the code.
The logic was derived from FlowRunner where also the name is empty.

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java [hunk 2]
```diff
@@ -431,8 +433,8 @@ public LogData readFlowLogs(final int execId, final int startByte, final int len
     }
 
     try {
-      final File logFile = this.flowRunner.getFlowLogFile();
-      if (logFile != null && logFile.exists()) {
+      final File logFile = FlowContainer.logFile;
+      if (logFile.exists()) {
         return FileIOUtils.readUtf8File(logFile, startByte, length);
       } else {
         logger.warn("Flow log file does not exist for flow execId: {}", execId);
```
Comment (jakhani) [inline azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java:440]: Shouldn't this be an error?

File: azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java [hunk 3]
```diff
@@ -635,19 +637,32 @@ public static void deleteSymlinkedFile(final Path symlinkedFilePath)
...
 
+  /**
+   * Uploads the log file to the db for persistence.
+   * @param execId execution id of the flow.
+   */
+  private void uploadLogFile(final int execId) {
+    try {
+      this.executorLoader.uploadLogFile(execId, "", 0, FlowContainer.logFile);
+    } catch (final ExecutorManagerException e) {
+      e.printStackTrace();
```
Comment (jakhani) [inline azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java:648]: Can you please log the exception?

**Pull request merged**